torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -355,6 +357,16 @@ class TORCH_CUDA_CU_API TensorView : public Val {
     return axes_to_swizzle_;
   }
 
+  void gemmMMATile(const mma_utils::GemmTileParams& gemm_params);
```

Comment:
Please add comments here. In particular, I suppose there are some assumptions about this tensor. Is this intended to be only used from the `scheduleGemm` function? I think it'd be nice to have a manual option to use MMA without automatic scheduling.

Reply:
Yes I can add more comments here. The scheduler interface section of the design doc covers some of it. This utility has quite a few assumptions about the schedule and currently it's safer to be within scheduleGemm. Unfortunately mma ops are just hard to be made very universally compatible with arbitrary schedules due to the heavy assumptions made on thread data layout by the instructions themselves.

We will make it more flexible along the way for sure but if there're other cases that we wanted to extend to I can also prioritize.

Reply:
Maybe we could also have a fuser IR node that is pointwise mul + reduce, which could more directly map to an MMA. This is a common enough pattern seems it could be worth while. We just would want to make sure the broadcasting pattern is right for something like an MMA translation in fusion segmenter. Thinking along the lines of Welford.